contracts/token-distribution/SaleRounds.sol

```diff
@@ -151,6 +151,29 @@ contract SaleRounds is TokenDistribution, GameOwner, ERC20 {
         vestingStartTime = block.timestamp;
     }
 
+    function editReservedAllocations(string calldata _roundType, address _to, uint _amount) external
```

Comment:
This function overall isn't necessary, as it should be mappable with:
- removeAllocation
followed by
- reserveTokens

Reply:
Fixed